### PR TITLE
Add link to project

### PR DESCRIPTION
**Pull request by gbonnefille**
Useful to discover community.

Please ensure your pull request adheres to the following guidelines:
- [ ] explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [ ] Document your Changes in the `CHANGELOG.md` file of the respected chart as well as the `Chart.yaml`
- [ ] Bump the Version number of the respected chart

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,5 +7,6 @@
   <h1>Sonarsource Helm Charts Repo</h1>
   <p>Point Helm at this repository to access Sonarsource's helm charts</p>
   <p>helm repo add sonarqube https://SonarSource.github.io/helm-chart-sonarqube</p>
+  <p>Check sources on <a href="https://github.com/SonarSource/helm-chart-sonarqube">project<a>.</p>
```
Comment (leo-geoffroy-sonarsource) [inline index.html:10]: the <a> element is not closed correctly

**Comment by leo-geoffroy-sonarsource** [comment]
Hello @gbonnefille 

Thank you for the contribution, I left a comment on the PR

File: index.html (hunk 1 above)
Comment (gbonnefille) [reply index.html]: Oups... done.

**Comment by leo-geoffroy-sonarsource** [comment]
LGTM, thanks again @gbonnefille

**Pull request merged**